application-templates/django-ninja/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG MNP_UI
```

Comment:
Maybe there's a reason for the renaming but without any context on that my preference would be to keep the more generic name: CLOUDHARNESS_FRONTEND_BUILD instead

Reply:
Nah, this was a copy/paste from where we first implemented it in MNP, you're correct that the more generic name should be used.